test/setup/iam.tf

```diff
@@ -16,6 +16,7 @@
 
 locals {
   int_required_roles = [
+    "roles/cloudiot.admin",
```

Comment:
In my environment, the service account has the same roles. And the tests are passed. I'm not sure why 403 error happens when creating the iot registry.

Reply:
There is a possibility of the timing issue. Even if granting roles into IAM members, the roles can't be applied immediately.  I can see the same issue in my environment.
@aaron-lane Is this a known issue?

Reply:
Same issue: https://github.com/terraform-providers/terraform-provider-google/issues/3007
Please ignore this.

Reply:
Fixed in: https://github.com/terraform-google-modules/terraform-google-pubsub/pull/4/commits/61c18790863a5e922ba53ed5543ee7f59ea98c6e